### ISSUE TITLE
NumberFormatException: s == null

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-Android#17021]
**Sentry Url**: ~https://sentry.io/share/issue/8d8f044f0819433bab99b2d313e22937/~ https://a8c.sentry.io/share/issue/84d080a9ee5740c0baeadf2eb78e081d/
**User Count**: 54
**Count**: 2134
**First Release**: org.wordpress.android@15.1+885
**First Seen**: 2020-07-10T14:51:41.331000Z
**Last Seen**: 2022-08-09T10:11:54Z
**24 Hours**: 4
**30 Days**: 261

```
NumberFormatException: s == null
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.ImageBlockProcessor.processBlockJsonAttributes(ImageBlockProcessor.java:37)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.BlockProcessor.processBlock(BlockProcessor.java:91)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processBlock(MediaUploadCompletionProcessor.java:78)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processContent(MediaUploadCompletionProcessor.java:59)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processContent(MediaUploadCompletionProcessor.java:60)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processContent(MediaUploadCompletionProcessor.java:60)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processContent(MediaUploadCompletionProcessor.java:60)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processContent(MediaUploadCompletionProcessor.java:60)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processContent(MediaUploadCompletionProcessor.java:60)
     at org.wordpress.android.ui.posts.mediauploadcompletionprocessors.MediaUploadCompletionProcessor.processContent(MediaUploadCompletionProcessor.java:60)
...
(54 additional frame(s) were not displayed)
```

---

Not new (first seen 2y ago) but seing what looks like a peak recently (261 in last 30 days), so might be worth investigating further:
<img width="308" alt="image" src="https://user-images.githubusercontent.com/216089/183644460-b94bf3ba-b7c1-46fd-8df8-e27b92eb45cd.png">

**Comment by irfano** [comment]
Since it's a crash on the editing screen with a low impact, the priority is medium.

**Comment by irfano** [comment]
- The crash is probably caused by the null remote media id. 
 - It's worth to see @mkevins's [comment](https://github.com/wordpress-mobile/WordPress-Android/issues/12228#issuecomment-647249169) in a possible dupe [issue](https://github.com/wordpress-mobile/WordPress-Android/issues/12228).
 - A common error log on issues `i/WordPress-STATS: 🔵 Tracked: notification_shown, Properties: {"notification_type":"post_upload_error","default_tab_experiment":"dashboard"}` shows the crash may occur after the post upload error.
 - This [PR](https://github.com/wordpress-mobile/WordPress-Android/pull/13661) handles possible remote `mediaId` case. If solving the root problem is challenging, handling the crash in a similar way could be considered.

**Comment by mkevins** [comment]
I have not been able to reproduce this, but I think this:

> A common error log on issues i/WordPress-STATS: 🔵 Tracked: notification_shown, Properties: {"notification_type":"post_upload_error","default_tab_experiment":"dashboard"} shows the crash may occur after the post upload error.

is a good hint. Also, I have flipped through a few of the events (but not an exhaustive search), and out of the the ones I observed, the crash occurs with `Image` blocks. That could simply be because those are most often used (gallery also using Image blocks internally), or possibly the steps to reproduce may require the use of one or more image blocks. If there are cases where this crash occurs with other media containing blocks, it might point to something more general.

I have drafted a PR here: https://github.com/wordpress-mobile/WordPress-Android/pull/17089 which follows the same approach in this PR: https://github.com/wordpress-mobile/WordPress-Android/pull/13661.
